atd-vze/src/Components/GridDateRange.js

```diff
@@ -114,6 +113,7 @@ const GridDateRange = ({
           // Prevent user from selecting start date after current date
           maxDate={maxDate}
           minDate={minDate}
+          strictParsing={true}
```

Comment:
fixes some weird behavior where if you start erasing with your keyboard `onChange()` will fire not just once you've completely emptied the textbox but at various intervals, for example if the date was "12/20/2020" and you start erasing the characters once you get to "1" `onChange()` fires and sets the date to "01/01/2024".

Reply:
ah, interesting that this component supports this behavior!

i was _just_ dealing with this in Moped, and ended up writing [this handler](https://github.com/cityofaustin/atd-moped/blob/98892dc964d8fa5630c69c7e62df5536056e4f8c/moped-editor/src/components/forms/ControlledDateField.js#L5C1-L13C3) to test when a date was valid, because sometimes it fires off random partial date strings that throw an error.

Reply:
@johnclary yeah this was super convenient and i only figured out about this prop by combing thru their github issues :sweat_smile:

Reply:
🕵🏼 nice find!